app/services/post_check_availability_task_service.rb

```diff
@@ -21,6 +21,11 @@ def validate_options
     raise("Options must have task key") if @options[:task].blank?
   end
 
+  def ok_services
+    TaskRetryService.new(:source_id => @source.id).process
```

Comment:
@hsong-rh Since these 2 are independent activities not dependent on each other if one fails the other one is not going to run. Should we have a try/catch around each one of them so they are independently executed